.gitignore

```diff
@@ -1,8 +1,160 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This is a traditional `.gitignore` for `python`
https://github.com/github/gitignore/blob/main/Python.gitignore